### PR TITLE
first proposal of CV32E40Pv2 ci_check script

### PR DESCRIPTION
**Pull request by XavierAubert**
Here is a first proposal for a ci_check script. 

I added all configurations from `cv32e40p/tests/cfg`, and mixed them a bit depending on what it seems to be relevant for me. 

Don't hesitate to tell me if anything's missing/wrong

Some tests are commented because they are not already merged in our working branch (`cv32e40p/dev`) and it will cause the cv_regress to issue a critical error. 

The commands to generate & run the ci_check script will be, for running with vsim and Questa Verification Run Manager with 4 jobs in parallel : 
```shell
./cv_regress --rmdb --file=cv32e40pv2_ci_check.yaml --simulator=vsim --outfile=cv32e40pv2_ci_check.rmdb --lsf "bsub -K -q 'vanilla'" --check_sim_results
vrun -rmdb cv32e40pv2_ci_check.rmdb -run cv32e40p -j 4 -mintimeout 0:300 -nolocalrerun
```

To generate a shell script, the command is : 
```shell
./cv_regress --sh --file=cv32e40pv2_ci_check.yaml --simulator=vsim --outfile=cv32e40pv2_ci_check.sh
```

### CODE REVIEW

File: cv32e40p/regress/cv32e40pv2_ci_check.yaml [hunk 1]
```diff
@@ -0,0 +1,334 @@
...
+    cmd: make comp
+    cfg: pulp_fpu_2cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_fpu_3cyclat:
+    cmd: make comp
+    cfg: pulp_fpu_3cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_fpu_4cyclat:
```
Comment (pascalgouedo) [inline cv32e40p/regress/cv32e40pv2_ci_check.yaml:52]: 4 cycle latency are going to be removed.

File: cv32e40p/regress/cv32e40pv2_ci_check.yaml [hunk 2]
```diff
@@ -0,0 +1,334 @@
...
+    cmd: make comp
+    cfg: pulp_fpu_zfinx_2cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_fpu_zfinx_3cyclat:
+    cmd: make comp
+    cfg: pulp_fpu_zfinx_3cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_fpu_zfinx_4cyclat:
```
Comment (pascalgouedo) [inline cv32e40p/regress/cv32e40pv2_ci_check.yaml:77]: Same

File: cv32e40p/regress/cv32e40pv2_ci_check.yaml [hunk 3]
```diff
@@ -0,0 +1,334 @@
...
+    cmd: make comp
+    cfg: pulp_cluster_fpu_2cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_cluster_fpu_3cyclat:
+    cmd: make comp
+    cfg: pulp_cluster_fpu_3cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_cluster_fpu_4cyclat:
```
Comment (pascalgouedo) [inline cv32e40p/regress/cv32e40pv2_ci_check.yaml:107]: Same

File: cv32e40p/regress/cv32e40pv2_ci_check.yaml [hunk 4]
```diff
@@ -0,0 +1,334 @@
...
+    cmd: make comp
+    cfg: pulp_cluster_fpu_zfinx_2cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_cluster_fpu_zfinx_3cyclat:
+    cmd: make comp
+    cfg: pulp_cluster_fpu_zfinx_3cyclat
+    dir: cv32e40p/sim/uvmt
+
+  uvmt_cv32e40p_pulp_cluster_fpu_zfinx_4cyclat:
```
Comment (pascalgouedo) [inline cv32e40p/regress/cv32e40pv2_ci_check.yaml:132]: Same

**Review by pascalgouedo — CHANGES_REQUESTED** [review]
LGTM

**Review by dd-vaibhavjain — APPROVED** [review]

**Comment by MikeOpenHWGroup** [comment]
LGTM as well.  It seems all the comments were resolved and both @pascalgouedo and @dd-vaibhavjain have approved this PR, so I will merge it in.

**Pull request merged**